tests/utils/test_geometry_comparisons.py

```diff
@@ -25,33 +25,46 @@ def test_compare_poses_with_uniform_scaled_translations(self):
         """Check pose comparison with all translations in input #2 scaled by
         the same scalar factor."""
         scale_factor = 1.2
-        pose_list_ = [Pose3(x.rotation(), x.translation() * scale_factor) for x in POSE_LIST]
+        pose_list_ = [
```

Comment:
i forgot to mention we are using `black myfile.py -l 120` to allow for a 120-line limit, instead of the default 88, since we thought it made things a bit more readable and less choppy

Reply:
Sure, I can change that. But I should mention that the 80 character limit is actually based on a lot of discussion on readability and is widely accepted. 
For more details: https://www.python.org/dev/peps/pep-0008/#maximum-line-length
https://google.github.io/styleguide/pyguide.html#32-line-length